pyrobuf/protobuf/templates/proto_pyx.tmpl

```diff
@@ -8,21 +8,45 @@ from pyrobuf_util cimport *
 import base64
 import json
 import warnings
+import enum
 
 {%- for import in imports %}
 from {{ import }}_proto cimport *
 {%- endfor %}
 
-{%- macro message_enum_fields_def(enum) %}
-    {%- for field in enum.fields.values() %}
-    {{ field.name }} = _{{ field.full_name }}
+
+{%- macro enum_fields_def(enum) %}
+class {{enum.full_name}}(enum.Enum):   
```

Comment:
I'm a little wary of subclassing `Enum` here because I think `Enum` values are a special type - will need to confirm unit tests all still pass.